pages/en/lb4/Routes.md

```diff
@@ -96,9 +96,8 @@ function greet(name: string) {
 }
 
 (async function start() {
-  const server = await app.getServer(RestServer);
   const route = new Route('get', '/', spec, greet);
-  server.route(route);
+  app.route(route); // attaches route to RestServer
```

Comment:
Please move route registration out of async `start()` function.

I think we no longer need async start function and can call app.start() directly:

```ts
// greet is a basic operation
 function greet(name: string) {
   return `hello ${name}`;
 }

const route = new Route('get', '/', spec, greet);
app.route(route);

app.start();
```